File: launchdarkly/ruby/src/PatchTeamExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

team_patch_input = LaunchDarklyClient::TeamPatchInput.new
team_patch_input.instructions = JSON.parse(<<-EOD
    [
        {
            "kind": "updateDescription",
            "value": "New description for the team"
        }
    ]
    EOD
)
team_patch_input.comment = "Optional comment about the update"

begin
    response = LaunchDarklyClient::TeamsApi.new.patch_team(
        "teamKey_string", # team_key
        team_patch_input,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling TeamsApi#patch_team: #{e}"
end

File: launchdarkly/python/src/PatchTeamExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    team_patch_input = models.TeamPatchInput(
        instructions=json.loads("""
            [
                {
                    "kind": "updateDescription",
                    "value": "New description for the team"
                }
            ]
        """),
        comment="Optional comment about the update",
    )

    try:
        response = api.TeamsApi(api_client).patch_team(
            team_key=None,
            team_patch_input=team_patch_input,
            expand=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Teams#patch_team: %s\n" % e)
